sql/catalyst/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -64,6 +69,13 @@ sealed abstract class Filter {
   private[sql] def containsNestedColumn: Boolean = {
     this.v2references.exists(_.length > 1)
   }
+
+  /**
+   * Converts V1 filter to V2 filter
+   *
+   * @since 3.3.0
```

Comment:
we should remove `since` as it's not a public API

Reply:
@beliefer

Reply:
I see.